src/types/events/base-events.ts

```diff
@@ -898,28 +897,3 @@ export interface WorkflowStepExecuteEvent {
 
 // NOTE: `user_resourced_denied`, `user_resource_granted`, `user_resourced_removed` are left out because they are
 // deprecated as part of the Workspace Apps Developer Preview
-
-export interface FunctionExecutedEvent {
```

Comment:
Removed due to reverted commit included in history after bad rebase. The changed version should match `main`. To verify: [L899](https://github.com/slackapi/bolt-js/blob/237a503d1e19a8e40ca9ed130dacab1be9752d53/src/types/events/base-events.ts#L899)